lang/src/org/partiql/lang/errors/ErrorAndErrorContexts.kt

```diff
@@ -99,16 +98,16 @@ abstract class PropertyValue(val type: PropertyType) {
 
     val value: Any
         get() = when (type) {
-            LONG_CLASS      -> longValue()
-            STRING_CLASS    -> stringValue()
-            INTEGER_CLASS   -> integerValue()
-            TOKEN_CLASS     -> tokenTypeValue()
-            ION_VALUE_CLASS -> ionValue()
+            PropertyType.LONG_CLASS -> longValue()
+            PropertyType.STRING_CLASS    -> stringValue()
+            PropertyType.INTEGER_CLASS   -> integerValue()
+            PropertyType.TOKEN_CLASS     -> tokenTypeValue()
+            PropertyType.ION_VALUE_CLASS -> ionValue()
```

Comment:
nit (formatting that may be fixed by ktlint later): whitespace could be removed
```suggestion
            PropertyType.LONG_CLASS -> longValue()
            PropertyType.STRING_CLASS -> stringValue()
            PropertyType.INTEGER_CLASS -> integerValue()
            PropertyType.TOKEN_CLASS -> tokenTypeValue()
            PropertyType.ION_VALUE_CLASS -> ionValue()
```